code/modules/antagonists/wizard/equipment/soulstone.dm

```diff
@@ -88,6 +88,17 @@
 	one_use = TRUE
 	grab_sleeping = FALSE
 
+/obj/item/soulstone/anybody/sparring
+	icon_state = "purified_soulstone"
+	theme = THEME_HOLY
+	one_use = TRUE
+	grab_sleeping = FALSE
```

Comment:
can't you just make this a subtype of /obj/item/soulstone/anybody/chaplain?

Reply:
i should